### ISSUE TITLE
It's not excluding color-contrast checks

### ISSUE THREAD

**Issue by JohnSmall** [dequelabs/axe-matchers#4]
Given I have a step
"Then the page should be accessible according to: wcag2a, wcag2aa but skipping: color-contrast"
Then I get the response
"Elements must have sufficient color contrast:" and so on.

I see that it calls 

Invocation: axe.a11yCheck({}, {"runOnly":{"type":"tag","values":["wcag2a","wcag2aa"]},"rules":{"color-contrast":{"enabled":false}}}, callback); (RuntimeError)

So it's a problem in the axe.a11yCheck method.

**Comment by jasonkarns** [comment]
@dylanb Has this bug been reproduced directly with axe-core? can it be moved over to the axe-core repo?

**Comment by dylanb** [comment]
@jasonkarns I think this issue exists https://github.com/dequelabs/axe-core/issues/135

@WilcoFiers can you confirm this?

**Comment by WilcoFiers** [comment]
@dylanb Sorry, missed this one, late reply. But yes, this isn't supported currently, but like the comment on #135 says we've got this feature running internally. It'll be in the next major release, but we might want to release it as a minor update so people can get their hands on it earlier.

**Comment by JohnSmall** [comment]
@WilcoFiers as it's a minor bug fix not a new feature then a minor update release is appropriate. It would be nice to get the fix early.

**Comment by JohnSmall** [comment]
@WilcoFiers When is the minor update for axe-core due out so we can exclude color-contrast

**Comment by marcysutton** [comment]
We've done a few releases since the last comments on this issue, I would recommend trying again to see if it was fixed. aXe is now on version 2.1.7.

**Comment by NicholasEF** [comment]
Not sure if this is directly related, but I observed that:
`expect(page).to be_accessible.skipping :color-contrast`
doesn't work in Ruby 1.9+. The hyphen is the problem. It needs to be:
`:'color-contrast'`

**Comment by marcysutton** [comment]
It sounds like a solution was found by using `:color-contrast`, so I'm going to close this issue. Please let us know if you still have any problems.
